epistemix_platform/src/epistemix_platform/use_cases/upload_results.py

```
@@ -0,0 +1,160 @@
+    # 4. Get presigned S3 URL for upload
+    job_upload = JobUpload(
+        context="run",
+        upload_type="results",
+        job_id=job_id,
+        run_id=run_id,
+    )
+
+    try:
+        upload_location = upload_location_repository.get_upload_location(job_upload)
+        logger.info(f"Got presigned URL for run {run_id} results upload")
+    except Exception as e:
```

Ruff (0.14.1)

Do not catch blind exception: `Exception`

(BLE001)